server/schedulers/evict_slow_store.go

```diff
@@ -175,36 +182,35 @@ func (s *evictSlowStoreScheduler) Schedule(cluster schedule.Cluster) []*operator
 		return ops
 	}
 
-	slowStores := make([]*core.StoreInfo, 0)
+	var slowStore *core.StoreInfo
+
 	for _, store := range cluster.GetStores() {
 		if store.IsRemoved() {
 			continue
 		}
 
 		if (store.IsPreparing() || store.IsServing()) && store.IsSlow() {
-			slowStores = append(slowStores, store)
+			// If there is more than onw slow store do nothing.
+			if slowStore != nil {
```

Comment:
should it report some metrics to analyze the reason why not generates operators?

Reply:
Maybe we could report the number of slow stores in another PR.